packages/bpk-mixins/src/mixins/_svgs.scss

```diff
@@ -193,17 +193,10 @@
 ///     @include bpk-spinner(spinner-sky-gray-tint-02);
 ///   }
 
-@mixin bpk-spinner($spinner: '') {
+@mixin bpk-spinner() {
   width: bpk-spacing-base();
   height: bpk-spacing-base();
   animation: bpk-keyframe-spin 600ms linear infinite;
-
-  @if $spinner != '' {
```

Comment:
This can safely be removed as doing research on its usage only one project uses this and they are way behind on Backpack versioning already